app/controllers/internal/events_controller.rb

```diff
@@ -54,6 +55,10 @@ def edit
 
   private
 
+    def set_event_type(type)
```

Comment:
First time I read this I thought the method was going to assign something. How about something like:

```
def determine_event_type
  event_types[params.fetch(:event_type, :provider)]
end
```